src/Objs/Homepage/SiteColorsPicker.tsx

```diff
@@ -43,6 +43,45 @@ export const SiteColorsPicker = connect(function SiteColorsPicker({
           />
         </div>
       </div>
+
+      <div className="row">
+        <div className="col-sm-4">
+          <div className="scrivito_detail_label">
+            <span>Light grey</span>
```

Comment:
* [ ] FYI I see that we will change the names in a next step and there is a confusing in-between state. To reduce confusion in this dialog, how about going with the future names from the start? It looks weird to have a color selector for "grey". Idea:
* Light backdrop
* Medium backdrop
* Dark backdrop

Reply:
> I see that we will change the names in a next step [...]

Which next step are you referring two?

Reply:
E.g. `siteColorLightGrey` -> `siteColorLightBackdrop`